vgstation13.dme

```diff
@@ -2298,7 +2299,7 @@
 #include "interface\web\interface.dms"
 #include "maprendering\maprendering.dm"
 #include "maps\_map.dm"
-#include "maps\tgstation.dm"
+#include "maps\test_tiny.dm"
```

Comment:
please keep this i hate changing it to test_tiny

Reply:
every time